From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Tue, 17 Sep 2019 15:38:05 -0700
Subject: ANDROID: init: GKI: enable hidden configs for regmap

Add hidden configs to GKI_HACKS_TO_FIX so they are
enabled for loadable regmap modules built out-of-tree.

Bug: 141266428
Change-Id: I1dedb1fd6e26e36c12b28fcbbf0302f074547101
Signed-off-by: Todd Kjos <tkjos@google.com>
---
 init/Kconfig.gki | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/init/Kconfig.gki b/init/Kconfig.gki
index 6c3f89bd8e25..45966f3325b9 100644
--- a/init/Kconfig.gki
+++ b/init/Kconfig.gki
@@ -9,6 +9,14 @@ config GKI_HIDDEN_DRM_CONFIGS
 	  These are normally selected implicitely when including a
 	  DRM module, but for GKI, the modules are built out-of-tree.
 
+config GKI_HIDDEN_REGMAP_CONFIGS
+	bool "Hidden Regmap configs needed for GKI"
+	select REGMAP_IRQ
+	help
+	  Dummy config option used to enable hidden regmap configs.
+	  These are normally selected implicitely when a module
+	  that relies on it is configured.
+
 # Atrocities needed for
 # a) building GKI modules in separate tree, or
 # b) building drivers that are not modularizable
@@ -19,6 +27,7 @@ config GKI_HIDDEN_DRM_CONFIGS
 config GKI_HACKS_TO_FIX
 	bool "GKI Dummy config options"
 	select GKI_HIDDEN_DRM_CONFIGS
+	select GKI_HIDDEN_REGMAP_CONFIGS
 	help
 	  Dummy config option used to enable core functionality used by
 	  modules that may not be selectable in this config.
